rpc/backend/account_info.go

```diff
@@ -48,7 +48,7 @@ func (b *Backend) GetProof(address common.Address, storageKeys []string, blockNr
 		return nil, err
 	}
 
-	height := int64(blockNum)
+	height := blockNum.Int64()
```

Comment:
Let's refactor the Int64() function to be named AdjustedInt64() to clearly denote that it transforms 0's into 1's. Otherwise, PR looks good to me!